File: MultiplesOf3or5/solution.go
package main

import "fmt"

func sumOfMultiplesOf3or5(limit int) int {
	sum := 0

	for i := 0; i < limit; i++ {
		if i%3 == 0 || i%5 == 0 {
			sum += i
		}
	}
	return sum
}

func main() {
	fmt.Println(sumOfMultiplesOf3or5(1000))
}

File: MultiplesOf3or5/solution.py
from itertools import accumulate


def sum_of_multiples_of_3or5(limit):
    return sum([x for x in range(limit) if x % 3 == 0 or x % 5 == 0])

if __name__ == "__main__":
    limit = 1000
    print(sum_of_multiples_of_3or5(limit))